docs/getting-started.md

```diff
@@ -44,6 +44,12 @@ Finally, check if TopoLVM is running. All TopoLVM pods should be running.
 kubectl get pod -n topolvm-system
 ```
 
+## Configure node startup taint
+
+There are potential race conditions on node startup (especially when a node is first joining the cluster) where pods/processes that rely on the a CSI Driver can act on a node before the CSI Driver is able to startup up and become fully ready. To combat this, the TopoLVM contains a feature to automatically remove a taint from the node on startup. Users can taint their nodes when they join the cluster and/or on startup, to prevent other pods from running and/or being scheduled on the node prior to the CSI Driver becoming ready.
```

Comment:
I guess you copied these code from [aws-ebs-csi-driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) project. Am I right?

In that case, please replace the contents of this section with the link to the original document.

(I guess [this one](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/5a56f6672dc474604bb90d65a3fc067560cf6c1b/docs/install.md#configure-node-startup-taint) is the original.)

Reply:
sure I will probably have time over the weekend. I may go ahead with a new PR on this so I can compare the two functionally